dependencies/HaXml-1.25.3/src/Text/XML/HaXml/Wrappers.hs

```diff
@@ -31,7 +31,7 @@ fix2Args :: IO (String,String)
 fix2Args = do
   args <- getArgs
   when ("--version" `elem` args) $ do
-      putStrLn $ "part of HaXml-"++ VERSION
+      putStrLn $ "part of HaXml-"
```

Comment:
@R0ss These changes don't belong in this pull request; if you want, you can submit them separately.